resources/icons/favorite-star.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path fill="var(--favorite-star-bg, none)" stroke="var(--favorite-star-outline, currentColor)" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" d="m10.558 1.411 2.711 5.273 5.12.497c.603.1.804.697.402 1.094l-4.217 4.179 1.607 5.67c.1.498-.402 1.094-1.004.796l-5.221-2.686-5.12 2.587c-.302.198-1.105-.1-.904-.796l1.606-5.77-4.317-3.98c-.3-.397-.401-.994.402-1.094l5.12-.497L9.454 1.41c.301-.597.803-.497 1.104 0h0z"/></svg>
```

Comment:
do we not have a star icon in the foundation lib?

Reply:
We do but it doesn't allow changing the inner background which is required for selected favourites